File: Data-Structures-DSA/CPP/C++/1.py
n = int(input())
# for i in range(1,n):
#     print("*"*(2*i-1))
for i in range(1,n+1):
    if i ==1 or i == n:
        print("*"*n)
    else:
        print("*" + " "*(n-2) + "*")

File: Web-Development/Practice-JS/1.js
// Aapka data (string ya array)
const data = "0000001111111";
const getOneIndices = (input) => {
  const indices = [];
  
  // Ek simple loop jo bilkul natural lagta hai
  for (let i = 0; i < input.length; i++) {
    if (input[i] === '1') {
      indices.push(i);
    }
  }
  
  return indices;
};
const result = getOneIndices(data);
console.log("Indices of 1:", result);



const data = "0000001111111";
const indices = [...data].reduce((acc, char, index) => {
  if (char === '1') acc.push(index);
  return acc;
}, []);

console.log(indices);